alacritty/src/logging.rs

```diff
@@ -122,13 +125,31 @@ impl log::Log for Logger {
 
         // Write to stdout.
         if let Ok(mut stdout) = self.stdout.lock() {
-            let _ = stdout.write_all(msg.as_ref());
+            let _ = stdout.write_all(msg);
         }
     }
 
     fn flush(&self) {}
 }
 
+fn create_log_message(record: &log::Record<'_>, target: &str) -> String {
+    let now = time::strftime("%F %T.%f", &time::now()).unwrap();
+    let mut msg = format!("[{}] [{:<5}] [{}] ", now, record.level(), target);
+
+    // Alignment for the lines after the first new line character in the payload. We don't deal
+    // with fullwidth/unicode chars here, so just `msg.len()` will work fine.
+    let alignment = msg.len();
+
+    // Push lines with adding extra padding into the next line, we'll trim it later on.
```

Comment:
```suggestion
    // Push lines with extra padding on the next line, which is trimmed later.
```

Commenting this on the trimming function might also be an option.